docs/pages/repo/docs/reference/command-line-reference/run.mdx

```diff
@@ -131,9 +131,11 @@ for execution.
 
 Multiple filters can be combined to select distinct sets of targets. Additionally, filters
 can also exclude targets. A target that matches any filter and is not explicitly excluded will
-be in the final entrypoint selection.
+be in the final entrypoint selection. In other words, the final list of targets is an _union_
+of all the targets matching the filters provided, not an intersection.
```

Comment:
Not quite technically correct. It's the union of the inclusions, minus the union of the exclusions.

```suggestion
of all the targets matching the filters provided, minus the union of targets that have been excluded.
```